src/ecrc-frontend/src/components/page/orgVerification/OrgVerification.js

```diff
@@ -10,21 +10,22 @@ import Table from "../../composite/table/Table";
 import SideCards from "../../composite/sideCards/SideCards";
 
 export default function OrgVerification({ page: { header, org } }) {
-  const history = useHistory();
+  const [toHome, setToHome] = useState(false);
+  const [toTOU, setToTOU] = useState(false);
 
   useEffect(() => {
     window.scrollTo(0, 0);
     if (!org.orgNm) {
-      history.push("/");
+      setToHome(true);
     }
-  }, [history, org.orgNm]);
+  }, [org.orgNm]);
 
   const orgVerified = () => {
-    history.push("/ecrc/termsofuse");
+    setToTOU(true);
```

Comment:
why not just do the redirects here instead of setting state and then doing it inside the render function?

Reply:
You could, I just thought it looked odd having a component return there.

Reply:
I think that is cleaner than having ternary or conditional checks in the render method